### PR TITLE
Changelog 06192021

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
Update the changelog (hopefully for the last time in the 0.F release cycle).

#### Describe the solution
tools/generate_changelog.py -f --by-date changelog_06 -t ../repo_token -e 2021-06-19 2021-04-15
tac changelog_06 > changelog_06_rev
tools/merge_summaries.sh changelog_06_rev data/changelog.txt
Then copyedit the results.

#### Additional context
Typos happen, and some of the things I prune might be bigger than I remember or part of some larger set of changes that should go in the changelog, please take a look at not just the final diff, but also the copyedit commit to check for missed entries.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -114,6 +114,8 @@ Adds different bleeding types for different species.
 Adds bleeding butchery action.
 Autodoc installation programs added that almost guarantee flawless installation of supported bionics.
 Removes the obsolete 'carrion spawn rate scaling factor' world configuration option.
+Make thrown glass items drop glass shards when shattered.
+Adding ability to cranes (excluding tiny) and forklift arms to suspend creatures for butchering.
```
Comment (anothersimulacrum) [inline data/changelog.txt:118]:
```suggestion
Add ability to cranes (excluding tiny ones) and forklift arms to suspend creatures for butchering.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
 Avoids doing character body updates more than once per turn
 When harvesting crops, drops the result on the player's square.
 Improves road system connectivity across overmaps.
+Stop failures clustering at high completion percentages.
```
Comment (anothersimulacrum) [inline data/changelog.txt:353]:
```suggestion
Stop crafting failures clustering at high completion percentages.
```

File: data/changelog.txt [hunk 3]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Fix not-spring eternal season.
```
Comment (anothersimulacrum) [inline data/changelog.txt:354]:
```suggestion
Fix non-spring eternal seasons.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -215,6 +217,7 @@ Cap monsters at 1000 kg so butchering kind of works.
 [Magiclysm] Adds Workshop Elves, Yule Cats, and Leprechauns.
 [Aftershock] Adds Formless Ruins - a new aboveground, low-risk dungeon type.
 [Dark Skies Above] DSA is reborn as an actively-developed, total-conversion mod.
+[Many Mods] Fixes problematic faction relationships (one-way aggression).
```
Comment (actual-nh) [inline data/changelog.txt:220]:
```suggestion
[Vanilla + Many Mods] Fixes problematic faction relationships (one-way aggression).
```

File: data/changelog.txt [hunk 5]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
 Avoids doing character body updates more than once per turn
 When harvesting crops, drops the result on the player's square.
 Improves road system connectivity across overmaps.
+Stop failures clustering at high completion percentages.
+Fix not-spring eternal season.
+Stop unreloadable items from being reloadable.
+Fix savegame corruption from teleporting to very distant submaps
```
Comment (actual-nh) [inline data/changelog.txt:356]:
```suggestion
Fix savegame corruption causing teleporting to very distant submaps
```

File: data/changelog.txt [hunk 6]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Unfreeze trap-avoiding monsters.
+Fix ramming at low speed.
+Fix the crafting success formula when crafter is missing proficiencies.
```
Comment (actual-nh) [inline data/changelog.txt:359]:
```suggestion
Fix the crafting success formula when the crafter is missing proficiencies.
```

File: data/changelog.txt [hunk 7]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Placing specials around cities now scales with city size.
+Fix proficiency books' effect on crafting.
+Construction of roofs actually makes tile above the construction a roof.
+Partial vehicle damage reduction hotfix.
```
Comment (actual-nh) [inline data/changelog.txt:363]:
```suggestion
Construction of roofs actually makes the tile above the construction a roof.
Partial fix for vehicle damage reduction inconsistencies.
```

File: data/changelog.txt [hunk 8]
```diff
@@ -448,6 +463,10 @@ Reduces file size of composed tilesheets from compose.py.
 Implements C++ symbol demangling in crash reports on MacOS, MinGW, and Linux.
 Adds tests of automatic triggers for mutations.
 Generates a crash report on Android.
+Move Mingw cross-compile CI job from Travis to GitHub.
+compose.py: Fix --use-all when there are no filler sheets.
+compose.py: Don't warn about unused fillers matching IDs in tile entries from main sheets.
+Add a tool to check printf style format string in translations.
```
Comment (actual-nh) [inline data/changelog.txt:469]:
```suggestion
Adds a tool to check printf style format string in translations.
```

File: data/changelog.txt [hunk 9]
```diff
@@ -469,6 +488,8 @@ Adds a GitHub workflow running flake8 to check the Python scripts.
 Consistently styles code in CMakeLists.txt files.
 Adds CMake linting.
 Captures ODR violations using GOLD linker, ASAN, or LTO.
+Adds a CI action to build and release for Windows.
+Added fail-fast=false and android builds to Github actions.
```
Comment (actual-nh) [inline data/changelog.txt:492]:
```suggestion
Adds android builds to Github actions.
```

File: data/changelog.txt [hunk 10]
```diff
@@ -114,6 +114,8 @@ Adds different bleeding types for different species.
 Adds bleeding butchery action.
 Autodoc installation programs added that almost guarantee flawless installation of supported bionics.
 Removes the obsolete 'carrion spawn rate scaling factor' world configuration option.
+Make thrown glass items drop glass shards when shattered.
+Add ability to cranes (excluding tiny ones) and forklift arms to suspend creatures for butchering.
```
Comment (actual-nh) [inline data/changelog.txt:118]:
```suggestion
Makes thrown glass items drop glass shards when shattered.
Adds suspension for butchering by cranes (excluding tiny ones) and forklift arms.
```

**Review by actual-nh — COMMENTED** [review]
Looking good!

File: data/changelog.txt [hunk 11]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
 Avoids doing character body updates more than once per turn
 When harvesting crops, drops the result on the player's square.
 Improves road system connectivity across overmaps.
+Stop crafting failures clustering at high completion percentages.
```
Comment (pehamm) [inline data/changelog.txt:353]:
```suggestion
Stop crafting failures from clustering at high completion percentages.
```

File: data/changelog.txt [hunk 12]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Fix non-spring eternal seasons.
```
Comment (pehamm) [inline data/changelog.txt:354]:
```suggestion
Fixes non-spring eternal seasons.
```

File: data/changelog.txt [hunk 13]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Stop unreloadable items from being reloadable.
```
Comment (pehamm) [inline data/changelog.txt:355]:
```suggestion
Stops unreloadable items from being reloadable.
```

File: data/changelog.txt [hunk 14]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Fix savegame corruption from teleporting to very distant submaps
```
Comment (pehamm) [inline data/changelog.txt:356]:
```suggestion
Fixes savegame corruption from teleporting to very distant submaps
```

File: data/changelog.txt (hunk 5 above)
Comment (pehamm) [reply data/changelog.txt:356]:
```suggestion
Fixes savegame corruption causing teleporting to very distant submaps
```
For grammatical consistency

File: data/changelog.txt [hunk 15]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Unfreeze trap-avoiding monsters.
```
Comment (pehamm) [inline data/changelog.txt:357]:
```suggestion
Unfreezes trap-avoiding monsters.
```

File: data/changelog.txt [hunk 16]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Fix ramming at low speed.
```
Comment (pehamm) [inline data/changelog.txt:358]:
```suggestion
Fixes ramming at low speed.
```

File: data/changelog.txt [hunk 17]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Fix the crafting success formula when crafter is missing proficiencies.
+Placing specials around cities now scales with city size.
+Fix proficiency books' effect on crafting.
```
Comment (pehamm) [inline data/changelog.txt:361]:
```suggestion
Fixes proficiency books' effect on crafting.
```

File: data/changelog.txt [hunk 18]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
...
+Construction of roofs actually makes tile above the construction a roof.
+Partial vehicle damage reduction hotfix.
+Fix building ramps and stairs.
```
Comment (pehamm) [inline data/changelog.txt:364]:
```suggestion
Fixes building ramps and stairs.
```

File: data/changelog.txt [hunk 19]
```diff
@@ -448,6 +463,10 @@ Reduces file size of composed tilesheets from compose.py.
 Implements C++ symbol demangling in crash reports on MacOS, MinGW, and Linux.
 Adds tests of automatic triggers for mutations.
 Generates a crash report on Android.
+Move Mingw cross-compile CI job from Travis to GitHub.
```
Comment (pehamm) [inline data/changelog.txt:466]:
```suggestion
Moves Mingw cross-compile CI job from Travis to GitHub.
```

File: data/changelog.txt [hunk 20]
```diff
@@ -448,6 +463,10 @@ Reduces file size of composed tilesheets from compose.py.
...
+compose.py: Fix --use-all when there are no filler sheets.
```
Comment (pehamm) [inline data/changelog.txt:467]:
```suggestion
compose.py: Fixes --use-all when there are no filler sheets.
```

File: data/changelog.txt (hunk 6 above)
Comment (pehamm) [reply data/changelog.txt:359]:
```suggestion
Fixes the crafting success formula when the crafter is missing proficiencies.
```

**Review by pehamm — COMMENTED** [review]
Just some grammatical consistency details.

File: data/changelog.txt [hunk 21]
```diff
@@ -347,6 +350,18 @@ Prevents cross-z-level melee attacks when the target is not actually visible.
 Avoids doing character body updates more than once per turn
 When harvesting crops, drops the result on the player's square.
 Improves road system connectivity across overmaps.
+Stops crafting failures from clustering at high completion percentages.
+Fixes non-spring eternal seasons.
+Stops unreloadable items from being reloadable.
+Fixes savegame corruption causing teleporting to very distant submaps
+Unfreezes trap-avoiding monsters.
+Fixes ramming at low speed.
+Fixes the crafting success formula when the crafter is missing proficiencies.
```
Comment (I-am-Erk) [inline data/changelog.txt:359]: Can probably be left out, this fixes a bug that was not in last stable.
```suggestion
```

**Pull request merged**